File: m1-h1/src/main/java/ru/yegorpilipenko/otus/spring/m1h1/Application.java
package ru.yegorpilipenko.otus.spring.m1h1;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ImportResource;

import static org.springframework.boot.SpringApplication.run;

@SpringBootApplication
@ImportResource("classpath:context.xml")
public class Application {

    public static void main(final String... args) {
        run(Application.class, args);
    }
}

File: m1-h6/src/main/kotlin/ru/yegorpilipenko/otus/spring/m1h6/Application.kt
package ru.yegorpilipenko.otus.spring.m1h6

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.transaction.annotation.EnableTransactionManagement

@SpringBootApplication
@EnableTransactionManagement
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
